app/client/reducers/userReducer.js

```diff
@@ -71,6 +75,27 @@ const editStatus = (state = initialEditStatusState, action) => {
   }
 };
 
+const initialPinStatus = {
+  isRequesting: false,
+  error: null,
+  success: false
+};
+const pinStatus = (state = initialPinStatus, action) => {
```

Comment:
all of these request/success flows also could use refactoring haha...

Reply:
added todo: https://trello.com/c/cGlzTuiu/137-refactor-web-reducers-sagas